Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/core/VersioningPolicy.java

```diff
@@ -67,5 +67,5 @@
      * action if config versions do not match. It will simply update config version
      * to defined one upon saving the file, and leave anything else as is.
      */
-    DISMISSIVE;
```

Comment:
Plase leave these. Though not neccessary for enums without constructor, there might be constructor someday.